projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cdc/b2b/b2b-scenarios.core-e2e-spec.ts

```diff
@@ -0,0 +1,61 @@
+import * as cdc from '../../../../helpers/vendor/cdc/cdc';
+
+describe('CDC B2B scenarios', () => {
+  Cypress.on('uncaught:exception', (err, runnable) => {
+    return false;
+  });
```

Comment:
CDC e2e are the only place where I can see `Cypress.on('uncaught:exception')` handler. Why is it so?

Note: we can move discussing this after 6.0

Reply:
just to handle any `uncaught exceptions` while cypress opens.
Removed it.